contracts/DrawBeacon.sol

```diff
@@ -191,7 +191,7 @@ contract DrawBeacon is IDrawBeacon, Ownable {
     }
 
     /// @inheritdoc IDrawBeacon
-    function calculateNextBeaconPeriodStartTime(uint256 currentTime)
+    function calculateNextBeaconPeriodStartTime(uint256 _time)
```

Comment:
We should also update the param name in the interface.
https://github.com/pooltogether/pooltogether-contract-tsunami/blob/4b3c3f066d556076e481c4d902fe64b820c3a75d/contracts/interfaces/IDrawBeacon.sol#L107